core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -389,11 +389,10 @@ private[spark] class TaskSchedulerImpl(
     // (taskId, stageId, stageAttemptId, accumUpdates)
     val accumUpdatesWithTaskIds: Array[(Long, Int, Int, Seq[AccumulableInfo])] = synchronized {
       accumUpdates.flatMap { case (id, updates) =>
-        // We should call `acc.value` here as we are at driver side now.  However, the RPC framework
+        // We call `acc.value` here as we are at driver side now.  However, the RPC framework
```

Comment:
The "However..." bit also sounds out of date. Is this problem actually no longer an issue? then this whole comment can be removed?